### ISSUE TITLE
game controls itself in battles in mother 3 

### ISSUE THREAD

**Issue by kamikazilucas** [nba-emu/NanoBoyAdvance#360]
when i enter a battle the game automatically presses the a button also in menus the a button sometimes dosent work but in normal gameplay its fine

**Comment by fleroviux** [comment]
Please show a screenshot of your input configuration (under Config -> Input -> Configure).
Also what version of the game is this. The English translation patch version 1.3? Or another one?

**Comment by kamikazilucas** [comment]
> Please show a screenshot of your input configuration (under Config -> Input -> Configure). Also what version of the game is this. The English translation patch version 1.3? Or another one?

english version 1.3 
![image](https://github.com/nba-emu/NanoBoyAdvance/assets/20354993/c6e35ee9-909d-43c0-bf29-f138f78ad6d1)
my controller config,

**Comment by kamikazilucas** [comment]
fyi it says none rn because i dont have my controller on but when i have it on it says xbox 360 0

**Comment by fleroviux** [comment]
This is likely an issue specific to the controller you are using. To confirm please check that the issue only exists when the controller is plugged in.

**Comment by kamikazilucas** [comment]
> This is likely an issue specific to the controller you are using. To confirm please check that the issue only exists when the controller is plugged in.

yes it only happens when using controller not keyboard, but why is that i dont really want to play the game with a keyboard

**Comment by fleroviux** [comment]
What is your exact controller model?

**Comment by kamikazilucas** [comment]
> What is your exact controller model?

i use wii u pro controller with wiinusoft
